Update intro_adhoc.rst (#200)
ansible/ansible-documentation: docs/docsite/rst/command_guide/intro_adhoc.rst

You can also filter this output to display only certain facts, see the :ref:`ansible.builtin.setup <setup_module>` module documentation for details.

Check mode
----------

> In check mode, Ansible will not make any changes to the remote machines. It will only print the commands that would be run if check mode was not enabled

Reviewer: ```suggestion
In check mode, Ansible does not make any changes to remote systems. Ansible prints the commands only. It does not run the commands.
```

.. code-block:: bash

>     $  ansible all -a "touch check.py" -C

Reviewer 2: Hmm, I think using the command module (the default module) for a check mode test is a bad idea, since that will cause the task to simply be skipped as the command module does not support check mode.

How about this instead
```suggestion
    $  ansible all -m copy -a "content=foo dest=/root/bar.txt" -C
```
?

> Enabling check mode < ``-C`` > in the above command means Ansible does not actually create the ``check.py`` file on any remote systems.

Reviewer 2: I would also mention the long version:
```suggestion
Enabling check mode (``-C`` or ``--check``) in the above command means Ansible does not actually create the ``/root/bar.txt`` file on any remote systems.
```

> Enabling check mode (``-C`` or ``--check``) in the above command means Ansible does not actually create the ``/root/bar.txt`` file on any remote systems.

Reviewer 2: Sorry, one last nick - in case the file already exists (with different content), it would update it instead of creating it :)
```suggestion
Enabling check mode (``-C`` or ``--check``) in the above command means Ansible does not actually create or update the ``/root/bar.txt`` file on any remote systems.
```

Author: Agreed. Committed the recommendations

> The above command would have created the file check.py on the remote machines, but it did not actually create the file because check mode < -C > was enabled.

Reviewer: ```suggestion
Enabling check mode < ``-C`` > in the above command means Ansible does not actually create the ``check.py`` file on any remote systems.
```
